src/app/descriptor-languages/Jupyter.ts

```diff
@@ -0,0 +1,40 @@
+import { ExtendedDescriptorLanguageBean } from 'app/entry/extendedDescriptorLanguage';
+import { SourceFile, ToolDescriptor, Workflow } from 'app/shared/swagger';
+import { Dockstore } from '../shared/dockstore.model';
+
+const JUPYTER_DOCUMENTATION_URL = Dockstore.DOCUMENTATION_URL + '/not-yet-created.html'; // TODO point at correct page
```

Comment:
I would having the real url, even if the page doesn't exist yet. Easier to only update one thing later